src/main/resources/edu/cornell/kfs/core/documentstore/InstitutionPreferences.json

```diff
@@ -683,6 +757,17 @@
         "documentTypeCode": "BRL",
         "linkType": "kfs"
       }, {
+        "label": "CG Reporting Code",
+        "link": "kr/lookup.do?methodToCall=start&businessObjectClassName=edu.cornell.kfs.coa.businessobject.ContractGrantReportingCode&docFormKey=88888888&hideReturnLink=true",
+        "linkType": "kfs",
+        "permission": {
+          "templateNamespace": "KR-SYS",
+          "templateName": "Initiate Document",
```

Comment:
This appears to be a link to a lookup screen, but the link is restricted based on a document-initiation permission instead of a lookup or use-screen one. Is this intentional?